common/changes/office-ui-fabric-react/kathayer-portcommandbarfixes_2018-05-03-17-07.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Would you mind renaming this file to 6_0_commandbarAccessibility?

Reply:
Also, add a corresponding entry to the 6_0 release notes.md file sitting in the root